src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -7097,7 +7100,8 @@ static void partList(XmlWriter& xml, Score* score, MxmlInstrumentMap& instrMap)
         if (!bracketFound && part->nstaves() > 1) {
             int number = findPartGroupNumber(partGroupEnd);
             if (number < MAX_PART_GROUPS) {
-                partGroupStart(xml, number + 1, BracketType::NO_BRACKET, false);
+                const BracketItem* bi = Factory::createBracketItem(score->dummy());
```

Comment:
Wouldn't this cause a kind of memory leak if you don't add `delete bi`?

Reply:
Thanks for pointing this out. Just my sloppiness, because usually only one is created for an export. But still no excuse for improper coding.